gulpfile.babel.js

```diff
@@ -61,9 +74,9 @@ function sass() {
 
 // Copy and compress images
 function images() {
-  return gulp.src('src/assets/img/*')
+  return gulp.src('src/assets/images/*')
```

Comment:
What's the driver behind changing directory names?  I'd rather maintain backwards consistency wherever possible